website_form_metadata/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,5 @@
+[ This file is optional, it should explain how to configure
```

Comment:
You don't need to include this explanatory texts.

Reply:
Ok, is it needed to configure some bot in the repo?

Reply:
No, this is done inside OCA server all nights in a process that scans all OCA repositories and update the needed things.

Reply:
Ok, done